src/Http/Http.Results/test/ResultsTests.cs

```diff
@@ -32,6 +32,22 @@ public void Accepted_WithUrlAndValue_ResultHasCorrectValues()
         Assert.Equal(uri, result.Location);
         Assert.Equal(value, result.Value);
     }
+    [Fact]
+    public void AcceptedOfT_WithUrlAndValue_ResultHasCorrectValues()
+    {
+        // Arrange
+        var uri = "https://example.org";
+        var value = new Todo(1);
+
+        // Act
+        Results.AcceptedAtRoute(uri, value:value);
```

Comment:
What's going on here?

Reply:
Oh! I left this line here from a local test. Thanks for catching it.